### PR TITLE
#181151230 Configure Authors Haven to use PostgreSQL as DB Engine

### PR DESCRIPTION
**Pull request by Victoria129**
####  What does this PR do?
- Configures Authors Haven to use PostgreSQL as DB Engine

####  Description of Task to be completed?
- Create PostgreSQL database
- Configure Authors Haven to use PostgreSQL as DB Engine

####  How should this be manually tested?
- A database with a database name is created; In this case, the database name used is "**Invictus**".
- By running **"psql -U postgres"**, provide a password if required, run "**postgres=# \c invictus;**", (invictus is the name of the database). In order to see the tables that have been created, run **"invictus=# \dt;"**.
 
####  Any background context you want to provide?
- No

####  What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/181151230

#### Screenshots (if appropriate)
image

image

####  Questions:
- None

[Start #181151230]

### CODE REVIEW

File: authors/settings.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'invictus',
+        'USER': 'postgres',
+        'PASSWORD': '@Wonder29',
+        'HOST': 'localhost',
+        'PORT': '5432',
+    }
+}
+
```
Comment (Emmanuel-Dominic) [inline authors/settings.py]: Nice work, though take time to learn about how to set environment variables with python.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Django==4.0.2
+django-cors-middleware==1.5.0
+django-extensions==3.1.5
+djangorestframework==3.13.1
+PyJWT==2.3.0
+pytz==2021.3
+six==1.16.0
+tzdata==2021.5
```
Comment (Emmanuel-Dominic) [inline requirements.txt]: I can't see the Postgres dependency/adapter listed here.

**Review by Emmanuel-Dominic — CHANGES_REQUESTED** [review]
Try to rebase with **develop**, to expect charges done in only two files ie settings.py and requirements.txt

**Review by Emmanuel-Dominic — APPROVED** [review]
LGTM

**Comment by Daniel-Kigozi** [comment]
Looks okay

**Review by Daniel-Kigozi — APPROVED** [review]

**Review by semitalatimothy — APPROVED** [review]

**Comment by bogerepau** [comment]
It is working fine

**Pull request merged**